cpp/src/centrality/betweenness_centrality.cu

```diff
@@ -369,12 +379,12 @@ template <typename VT, typename ET, typename WT, typename result_t>
 void BC<VT, ET, WT, result_t>::rescale_vertices_betweenness_centrality(result_t &rescale_factor,
```

Comment:
Better return std::tuple<result_t, bool> instead to be more functional.

Reply:
Updated in eab569b